src/routes/index.svelte

```diff
@@ -56,7 +59,11 @@
 
   <ExploreByTopic url="/categories" suggestions={indexPageSuggestions} />
   <hr class="component-margin--2" />
-  <ExploreByAreaComponent {autosuggestData} bind:userInputValue on:click={() => submitFunction(userInputValue)}
+  <ExploreByAreaComponent
+    {renderError}
+    autosuggestData={config.autosuggestData}
```

Comment:
Is the autosuggestData for the ExploreByAreaComponent always going to be the same data from the config?
If it is could it not be imported/loaded in ExploreByAreaComponent rather than passing it all the way down?

Reply:
Done.